config/settings.yml

```diff
@@ -20,7 +20,7 @@ cdxj_indexer:
   working_directory: /web-archiving-stacks/data/indexes/cdxj_working
   backup_directory: /web-archiving-stacks/data/indexes/cdxj_backup
   main_cdxj_file: /web-archiving-stacks/data/indexes/cdxj/level0.cdxj
-  url: http://localhost/was/cdx?url=
+  url: https://swap.stanford.edu/was/cdx
```

Comment:
Just making sure: is it cool to have local dev pointing at prod?

Reply:
Yes good catch. It now matches what is in `Settings.was_seed.wayback_uri`. Arguably we don't need both configuration options, but that's a refactor for another time I think.